containers/LocationMarker.js

```diff
@@ -39,23 +42,30 @@ class LocationMarker extends PureComponent {
         onDragStart={this.handleDragStart}
         onDrag={onDrag}
         onDragEnd={this.handleDragEnd}
-        ref={(marker) => {
-          this.marker = marker;
-        }}
       >
         <MapView.Callout>
           <PinCallout {...location} />
         </MapView.Callout>
-      </MapView.Marker.Animated>
+      </MapView.Marker>
     );
   }
 }
 
+LocationMarker.defaultProps = {
+  location: {
+    latitude: 0,
+    longitude: 0,
+    status: '',
+    created: 0,
+  },
+};
+
 LocationMarker.propTypes = {
+  locationKey: PropTypes.string.isRequired,
   location: PropTypes.shape({
```

Comment:
When does it not have `location`?

Reply:
I found that when the marker is being unrendered the location object becomes null and it throws an error saying the prototypes failed... Not exactly sure why though